Magento_Checkout/web/template/minicart/subtotal.html

```diff
@@ -1,14 +1,7 @@
-<!--
-/**
- * Copyright © 2013-2017 Magento, Inc. All rights reserved.
- * See COPYING.txt for license details.
- */
--->
-<div class="subtotal grid minicart__subtotal">
-    <span class="label grid--two-cols">
-        <!-- ko i18n: 'Cart Subtotal' --><!-- /ko -->
-    </span>
-
+<div class="minicart-content__summary">
+    <div class="minicart-content__summary-label subtotal">
+        <span class="price"><!-- ko i18n: 'Cart Subtotal' --><!-- /ko --></span>
```

Comment:
move `<!-- ko i18n: 'Cart Subtotal' --><!-- /ko -->` to new line

Reply:
Done